pages/api_integration-full-node-streaming.md

```diff
@@ -86,6 +92,20 @@ Response will contain a `oneof` field that contains either:
 		- Prices within a Match are matched at the maker order price.
 	- `fill_amounts` contains the absolute, total filled quantums of each order as stored in state.
 		- fill_amounts should be zipped together with the `orders` field. Both arrays should have the same length.
+- `StreamTakerOrder`
+  - Contains a oneof `TakerOrder` field which represents the order that entered the matching loop.
+    - Could be a regular order or a Liquidation Order.
+  - Contains a `StreamTakerOrderStatus` field which represents the status of a taker order after it has finished the matching loop.
+    - `OrderStatus` is a uint32 describing the result of the taker order matching. Only value `0` indicates success. Possible values found [here](https://github.com/dydxprotocol/v4-chain/blob/main/protocol/x/clob/types/orderbook.go#L118-L152).
+      - @jonfung to update to static link
+    - `RemainingQuantums` represents the remaining amount of non-matched quantums for the taker order.
+    - `OptimisticallyFilledQuantums` represents the number of quantums filled *during this matching loop*. It does not include quantums filled before this matching loop, if the order was a replacement order and was previously filled.
+- `StreamSubaccountUpdate`
+  - Contains a singular `SubaccountId` object to identify the subaccount.
+  - multiple `SubaccountPerpetualPosition`s to represent the perpetual positions of the subaccount.
+    - each `SubaccountPerpetualPosition` contains a perpetual id and the size of the position in base quantums.
+  - multiple `SubaccountAssetPosition`s to represent the asset positions of the subaccount.
+    - each `SubaccountAssetPosition` contains an asset id and the size of the position in base quantums.
```

Comment:
A lot of this seems redundant with the (very nice + informative) comments in the protos which are pasted below. No need for both?